### DISCUSSION TITLE
Copyright

### DISCUSSION THREAD

**Discussion by Yongsoo65** [community/community#143910] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I converted my face picture to an image by  Pytorch Hub program based on Linux.
Can I use it for my book cover commercially?

**Comment by swimmingbrain** [comment]
Hey Yongsoo! If you took the original photo of your face or have the rights to use it, you have the legal right to use the derivative image commercially, including for a book cover. Just make sure that the model you used does not impose limitations on commercial usage.

Hope this helps! 🧠

**Reply by Yongsoo65** [reply]
Thank you for your participation. Which do you mean the model I used is a person or the model I used is a particular program? 
I am curious Git Hub is related to Pytorch Hub. If they are same, does Pytorch Hub permit the commercial use of my image because this is Git Hub site?

**Reply by swimmingbrain** [reply]
PyTorch Hub does not automatically grant commercial use rights to models or data simply because it is hosted on GitHub. You would need to check the license associated with each specific model or repository on GitHub.
With model I mean the machine learning model that you're using for the conversion on Pytorch. :)

**Comment by mecodeatlas** [comment]
Hi @Yongsoo65,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Pytorch Discussion Forum](https://discuss.pytorch.org/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
